presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduIntegrationDecimalColumns.java

```diff
@@ -99,6 +103,13 @@ private void doTestCreateTableWithDecimalColumn(TestDecimal decimal)
         assertEquals(0, actual, 0.3 * Math.pow(0.1, decimal.scale), "p=" + decimal.precision + ",s=" + decimal.scale + " => " + actual + ",insert = " + insertValue);
     }
 
+    @AfterClass(alwaysRun = true)
+    public final void destroy()
+    {
+        getQueryRunner().close();
```

Comment:
This isn't needed as `AbstractTestQueryFramework` will close the returned query runner

Reply:
ping